fe/fe-core/src/main/java/com/starrocks/sql/analyzer/SelectAnalyzer.java

```diff
@@ -340,7 +340,7 @@ public void analyze(AnalyzeState analyzeState,
     }
 
     private void analyzeWindowFunctions(AnalyzeState analyzeState, List<Expr> outputExpressions,
-                                       List<Expr> orderByExpressions) {
+                                        List<Expr> orderByExpressions) {
```

Comment:
Why indent changed? Doesn't format check complains about it ?

Reply:
The previous code does not have automatic format, but this problem does not report an error